Upgrade WebsiteController for Symfony 6 compatibility (#6580)
sulu/sulu: UPGRADE.md

`Sulu\Bundle\WebsiteBundle\Controller\ExceptionController` / `sulu_website.exception_controller`
was removed. See also [UPGRADE 2.1.0-RC1](#210-rc1).

### WebsiteController methods removed and return types changed

> With support for Symfony 6 the `get` and `has` method to access services are deprecated and removed with Symfony 6.

Reviewer: ```suggestion
Symfony 6 has deprecated and removed the `get` and `has` methods to access services.
```

Instead, the methods from the container should be used:

```diff
-$this->has('twig');
-$this->get('twig');
+$this->container->has('twig');
+$this->container->get('twig');
```

> With the support for Symfony 6 the `getSubscribedServices` requires now an array return type:

Reviewer: ```suggestion
In order to support Symfony 6 the `getSubscribedServices` method requires an array return type:
```
